tools/review/get_comments_markdown.py

```
@@ -0,0 +1,296 @@
+
+def main(argv: List[str]) -> int:
+    # Basic flags
+    ap = argparse.ArgumentParser(description="Print PR comments as Markdown")
+    ap.add_argument("--owner", help="GitHub owner/org (optional)")
+    ap.add_argument("--repo", help="GitHub repo name (optional)")
+    ap.add_argument("--repo-slug", "--slug", dest="slug", help="GitHub repo as 'owner/repo'")
+    ap.add_argument("--remote", help="Explicit remote URL to parse for owner/repo")
+    ap.add_argument("--pr", type=int, help="Pull request number")
+    ap.add_argument("--out", help="Write output to file instead of stdout")
+    # Allow pr=123 shorthand
+    known, _ = ap.parse_known_args([])
```

Ruff (0.14.3)

Unpacked variable `known` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)